x-pack/solutions/observability/plugins/infra/common/constants.ts

```diff
@@ -52,6 +52,10 @@ export const EVENT_MODULE = 'event.module';
 export const METRICSET_MODULE = 'metricset.module';
 export const METRICSET_NAME = 'metricset.name';
 
+// OTel hostmetricsreceiver
+export const EVENT_DATASET = 'event.dataset';
```

Comment:
Would it be preferable to use `data_stream.dataset` instead here? That field should be present in all documents while `event.dataset` is added by one of our components. I think they should always be the same value.

Reply:
I think so.